src/routes/docs/products/ai/integrations/lmnt/+page.markdoc

```diff
@@ -0,0 +1,265 @@
+---
+layout: article
+title: Integrating LMNT
+description: Learn how to integrate LMNT into your Appwrite project.
+difficulty: intermediate
+readtime: 15
+---
+
+LMNT is a text-to-speech tool that can generate natural-sounding audio from text. It's an excellent tool for dubbing content, creating audiobooks, or even for accessibility.
+
+Integrating LMNT into your Appwrite project is simple. This tutorial will guide you through setting up the LMNT API and incorporating it into your Appwrite project.
+
+# Prerequisites {% #prerequisites %}
+
+- An Appwrite Project
+- An Appwrite Bucket
+- An [LMNT API Key](https://app.lmnt.com/account)
+
+{% section #step-1 step=1 title="Create new function" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console), click on **Functions** in the left sidebar and click the **Create Function** button.
+
+{% only_dark %}
+![Create function screen](/images/docs/functions/dark/template.png)
+{% /only_dark %}
+
+{% only_light %}
+![Create function screen](/images/docs/functions/template.png)
+{% /only_light %}
+
+1. In the Appwrite Console's sidebar, click **Functions**.
+2. Click **Create function**.
+3. Under **Connect Git repository**, select your provider.
+4. After connecting to GitHub, under **Quick start**, select the **Node.js** starter template.
+5. In the **Variables** step, add `APPWRITE_BUCKET_ID`, `LMNT_API_KEY`. Generate your LMNT Key [here](https://app.lmnt.com/account). For the `APPWRITE_API_KEY`, tick the box to **Generate API key on completion**.
```

Comment:
edit the together AI tutorial to talk about how the appwrite api key can be generated automatically